suite-native/intl/src/en.ts

```diff
@@ -449,6 +450,11 @@ export const en = {
         retrievengTakesLongerThanExpected:
             'Retrieving balances takes longer than usual. \n It may be caused by unstable internet connection.',
     },
+    passphrase: {
+        modal: {
+            enterWallet: 'Enter wallet',
```

Comment:
nitpick, I know it's not final, but:
```suggestion
            enterWallet: 'Enter passhprase',
```

Reply:
Yeah, I meant it like "enter into the wallet" :D But it's just a MVP copy, will change it, good point

Reply:
https://github.com/trezor/trezor-suite/pull/10703/commits/84efcd359621395ff1b61111420ba9d764c6c4f2